src/internal/modules/billing/controller.js

```diff
@@ -0,0 +1,176 @@
+const uuid = require('uuid/v4');
+const { selectBillingTypeForm, billingTypeFormSchema } = require('./forms/billing-type');
+const { selectBillingRegionForm, billingRegionFormSchema } = require('./forms/billing-region');
+const { viewBillRunListForm } = require('./forms/bill-run-list');
+const services = require('internal/lib/connectors/services');
+const forms = require('shared/lib/forms');
+const { get } = require('lodash');
+const moment = require('moment');
+const queryString = require('querystring');
+const helpers = require('@envage/water-abstraction-helpers');
+
+/**
+ * Step 1a of create billing batch flow - display form to select type
+ * i.e. Annual, Supplementary, Two-Part Tariff
+ * @param {*} request
+ * @param {*} h
+ */
+const getBillingBatchType = async (request, h) => {
+  const sessionForm = request.yar.get(get(request, 'query.form'));
```

Comment:
Might be nice to extract this to something like `getSessionForm(request)` and `clearSessionForm(request)`

Reply:
done and applied to both forms in this flow